modules/solid_mechanics/doc/content/source/materials/ComputeMultipleInelasticCosseratStress.md

```diff
@@ -1 +1,27 @@
-!template load file=stubs/moose_object.md.template name=ComputeMultipleInelasticCosseratStress syntax=/Materials/ComputeMultipleInelasticCosseratStress
\ No newline at end of file
+# ComputeMultipleInelasticCosseratStress
+
+!syntax description /Materials/ComputeMultipleInelasticCosseratStress
```

Comment:
There is a typo in here that shows up on the web page. `versions.Combinations` is missing a space. Not sure if you want to fix it now or later.

Reply:
fixed